lib/Test/Mojo.pm

```diff
@@ -17,7 +17,8 @@ use Mojo::Util qw(decode encode);
 use Test::More ();
 
 has [qw(message success tx)];
-has ua => sub { Mojo::UserAgent->new->ioloop(Mojo::IOLoop->singleton) };
+has ua  => sub { Mojo::UserAgent->new->ioloop(Mojo::IOLoop->singleton) };
+has dom => sub { shift->tx->res->dom };
```

Comment:
Yeah, the idea is that you can use Test::Mojo dom methods without doing a request by assigning a Mojo::DOM object, as illustrated by the changes in t/mojo/dom.t